manifests/install_integration.pp

```diff
@@ -0,0 +1,20 @@
+define datadog_agent::install_integration (
+  Enum['present', 'absent'] $ensure           = 'present',
+  String                    $integration_name = undef,
+  String                    $version          = undef,
+){
+
+  include datadog_agent
+
+  if $ensure == 'present' {
+    $command = 'install'
+  } else {
+    $command = 'remove'
+  }
+
+  exec { "${command} ${integration_name}==${version}":
+    command => "/opt/datadog-agent/bin/agent/agent integration ${command} ${integration_name}==${version}",
+    user    => 'dd-agent',
```

Comment:
This should probably require something like:
```
    require => Package[$datadog_agent::params::package_name],
```

We probably don't need to `notify`, we can leave that to the config file definition.

Reply:
👍